$NetBSD: patch-configure,v 1.20 2024/11/06 17:05:03 leot Exp $

Revert part of upstream commit f057de5a1a950a90d1920021db152a4b695f1a8a
and commit 3e7a6fbb8e38f930970a5dcc7a3f27cf6fdc04e8.

Avoid unconditional append of ldap and gssapi related .pc files to
libcurl.pc. Several platforms (e.g. NetBSD) do not provide such .pc and
their respective builtin.mk at the moment do not provide any logic to
generate fake .pc yet.

FIXME: This patch should be reverted and instead the corresponding builtin.mk
FIXME: files should be instructed to generate such .pc files.

--- configure.orig	2024-11-06 14:43:07.882717533 +0000
+++ configure
@@ -25042,7 +25042,6 @@ printf "%s\n" "no additional lib require
       else
         LIBS="$curl_cv_ldap_LIBS $curl_cv_save_LIBS"
       fi
-      LIBCURL_PC_REQUIRES_PRIVATE="ldap $LIBCURL_PC_REQUIRES_PRIVATE"
       { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $curl_cv_ldap_LIBS" >&5
 printf "%s\n" "$curl_cv_ldap_LIBS" >&6; }
       ;;
@@ -26189,13 +26188,6 @@ printf "%s\n" "found" >&6; }
         ;;
     esac
   fi
-  if test -n "$gnu_gss"; then
-    LIBCURL_PC_REQUIRES_PRIVATE="$LIBCURL_PC_REQUIRES_PRIVATE gss"
-  elif test "x$not_mit" = "x1"; then
-    LIBCURL_PC_REQUIRES_PRIVATE="$LIBCURL_PC_REQUIRES_PRIVATE heimdal-gssapi"
-  else
-    LIBCURL_PC_REQUIRES_PRIVATE="$LIBCURL_PC_REQUIRES_PRIVATE mit-krb5-gssapi"
-  fi
 else
   CPPFLAGS="$save_CPPFLAGS"
 fi
